src/hotspot/share/opto/loopnode.hpp

```diff
@@ -1387,7 +1387,14 @@ class PhaseIdealLoop : public PhaseTransform {
 
   // Is 'n' a (nested) member of 'loop'?
   int is_member( const IdealLoopTree *loop, Node *n ) const {
```

Comment:
Let's change this one as well while we're at it
```suggestion
  bool is_member( const IdealLoopTree *loop, Node *n ) const {
```